tensorflow/lite/micro/examples/network_tester/network_tester_test.cc

```diff
@@ -24,8 +24,8 @@ limitations under the License.
 #include "tensorflow/lite/schema/schema_generated.h"
 
 #ifdef ETHOS_U
-#include "tensorflow/lite/micro/examples/person_detection/person_image_data.h"
-#include "tensorflow/lite/micro/tools/make/downloads/person_model_int8/person_detect_model_data.h"
+#include "tensorflow/lite/micro/examples/person_detection/testdata/person_image_data.h"
```

Comment:
More of a question:  person_detection and network_tester both depends on this file. Is this more for convenience or are there other reasons?

Reply:
The network tester example uses the person detect model for the EthosU target. Its not a very good design to have one example reach into the other example's directory for input data. But it is what we have right now.

Once we clean up the examples at a high level with https://github.com/tensorflow/tflite-micro/issues/407, we should certainly revisit how we organize the data and models that are shared across examples.